src/Utils/Utils.jl

```diff
@@ -23,6 +23,8 @@ import CUDA  # To avoid name conflicts
 instantiate(x) = x
 instantiate(X::DataType) = X()
 
+getname(type) = typeof(type).name.wrapper
```

Comment:
Perhaps this should be called `getnamewrapper` (not sure what the difference between `name` and `name.wrapper` is)